Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'audited', '~> 5.0', '!= 5.1.0'
 gem 'will_paginate', '~> 3.3'
 gem 'ancestry', '~> 4.0'
 gem 'scoped_search', '>= 4.1.10', '< 5'
-gem 'ldap_fluff', '>= 0.7.0', '< 1.0'
+gem 'ldap_fluff', '>= 0.9.0', '< 1.0'
```

Comment:
Packaging was updated in https://github.com/theforeman/foreman-packaging/commit/e80816f710c8b6840ec5b1862bab9b5eabccfe19.